### DISCUSSION TITLE
Rounded table border

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#5398] [Feature requests]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

^7.3.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Can we define the radius for Table border?

![Screenshot 2023-12-08 at 22 10 24](https://github.com/mantinedev/mantine/assets/22228431/be26b6a2-de3e-491a-8e14-cff61ab8e324)

```
<Table withTableBorder withRowBorders={false}>
  {/* {...rows} */}
</Table>
```

I tried styling it with styles API (table, thead, tbody etc) but no luck.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by OrkhanAlikhanov** [comment]
Quick fix is to wrap inside `<Paper/>`:
```tsx
<Paper withBorder>
  <Table .../>
</Paper>
```
